src/hooks/help/useHelp.ts

```diff
@@ -35,6 +35,7 @@ const useHelpState = () =>
             case ReportType.BULLY:
             case ReportType.EMERGENCY:
             case ReportType.IM:
+            case ReportType.GUIDE:
                 newReport.reportedUserId = options.reportedUserId;
                 newReport.currentStep = ReportState.SELECT_CHATS;
```

Comment:
I think for a guide report we don't need to select any chats, maybe just skip to the step `ReportState.REPORT_SUMMARY` so the user can set the message that gets sent in the packet

Reply:
Made ! Thanks so much for the information ! 🙂